### ISSUE TITLE
Option to change the exchange rate provider

### ISSUE THREAD

**Issue by tadeubas** [BlueWallet/BlueWallet#5248]
BW is being used by lots of users to exchange sats for goods and sometimes they don't agree on the exchange rate of CoinDesk or (CoinDesk could be offline too?). It would be nice to give other options like CoinGecko.

The BTCPay server has a large list of exchange options available in its repo: https://github.com/btcpayserver/btcpayserver/blob/e27e93aa9a3e8102b3d9f272d93c0752d402b9d6/BTCPayServer.Rating/Services/RateProviderFactory.cs

**Comment by limpbrains** [comment]
Agree. Yestarday there was a 10% difference between coindesk and coinbase on BTC/CZK price. I think we should provide users an alternative source of exchange rates

**Comment by Overtorment** [comment]
i would just switch to smth else. maybe kraken price api.
too much hasle to allow switching providers

**Comment by Mark-Erik** [comment]
A switch to a more reliable exchange rate provider like Kraken would be greatly appreciated. 
As a trader, I lost an average of 3% with each euro denominated rate.
